nf_core/pipeline-template/.github/workflows/ci.yml

```diff
@@ -27,6 +29,20 @@ jobs:
         NXF_VER:
           - "24.04.2"
           - "latest-everything"
+        profile:
+          - "conda"
+          - "docker"
+          - "singularity"
+        test:
+          # TODO nf-core: You can customise CI pipeline run tests as required
```

Comment:
not sure if this should be a todo... I feel like better to document this somewhere instead of just having a todo item.

Reply:
It was a TODO before, I just moved the TODO around